### PR TITLE
search people with special caracter "-" and "é"

### PR DESCRIPTION
**Pull request by khouloudexo**

### CODE REVIEW

File: commons/src/main/java/org/exoplatform/platform/qa/ui/commons/Base.java [hunk 1]
```diff
@@ -82,10 +82,12 @@ public void afterEach() {
...
   }
 
+
   @AfterEach
   public void close() {
     if (!title().equals("Login")) {
       Selenide.close();
     }
   }
+
```
Comment (thomasdelhomenie) [inline commons/src/main/java/org/exoplatform/platform/qa/ui/commons/Base.java]: Useless changes

File: gatein/src/test/java/org/exoplatform/platform/qa/ui/gatein/GateinManagePagesTestIT.java [hunk 2]
```diff
@@ -120,6 +120,8 @@ public void test03_DeletePageOfPortal() {
      * click [Delete page] Input Data: Expected Outcome: - The page is removed from
      * the list
      */
+
+
```
Comment (thomasdelhomenie) [inline gatein/src/test/java/org/exoplatform/platform/qa/ui/gatein/GateinManagePagesTestIT.java]: Useless changes

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/plf/PlfPeopleSearchTestIT.java [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+import org.exoplatform.platform.qa.ui.commons.Base;
+import org.exoplatform.platform.qa.ui.gatein.pageobject.UserAddManagement;
+import org.exoplatform.platform.qa.ui.gatein.pageobject.UserAndGroupManagement;
+import org.exoplatform.platform.qa.ui.selenium.platform.HomePagePlatform;
+import org.exoplatform.platform.qa.ui.selenium.platform.NavigationToolbar;
+
+
+@Tag("gatein")
+@Tag("smoke")
+
```
Comment (thomasdelhomenie) [inline platform/src/test/java/org/exoplatform/platform/qa/ui/platform/plf/PlfPeopleSearchTestIT.java]: Blank line to remove

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/plf/PlfPeopleSearchTestIT.java [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+        info("Create new user");
+        useraddmanagement.addUser(username, password, email, firstName, lastName);
+        info("Search user on People");
+        homePagePlatform.goToConnections();
+        connectionsManagement.searchPeople(firstName, "","", "");
+        homePagePlatform.refreshUntil(firstName+" "+lastName,Condition.visible,1000);
+        info("Test Case 10: Delete user");
+        navigationToolbar.goToManageCommunity();
+        userandgroupmanagement.deleteUser(username);
+    }
```
Comment (thomasdelhomenie) [inline platform/src/test/java/org/exoplatform/platform/qa/ui/platform/plf/PlfPeopleSearchTestIT.java]: Indentation is not good

File: calendar/debug.log [hunk 5]
```diff
@@ -4,3 +4,15 @@
...
+[0319/113037.727:ERROR:prune_crash_reports.cc(34)] PruneCrashReportDatabase: Failed to get pending reports
+[0319/113101.409:ERROR:prune_crash_reports.cc(34)] PruneCrashReportDatabase: Failed to get pending reports
+[0319/113121.455:ERROR:prune_crash_reports.cc(34)] PruneCrashReportDatabase: Failed to get pending reports
+[0319/113203.701:ERROR:prune_crash_reports.cc(34)] PruneCrashReportDatabase: Failed to get pending reports
+[0319/113218.787:ERROR:prune_crash_reports.cc(34)] PruneCrashReportDatabase: Failed to get pending reports
+[0319/113239.116:ERROR:prune_crash_reports.cc(34)] PruneCrashReportDatabase: Failed to get pending reports
+[0319/121958.000:ERROR:prune_crash_reports.cc(34)] PruneCrashReportDatabase: Failed to get pending reports
+[0319/122200.400:ERROR:prune_crash_reports.cc(34)] PruneCrashReportDatabase: Failed to get pending reports
+[0319/122327.299:ERROR:prune_crash_reports.cc(34)] PruneCrashReportDatabase: Failed to get pending reports
+[0319/160538.800:ERROR:prune_crash_reports.cc(34)] PruneCrashReportDatabase: Failed to get pending reports
```
Comment (thomasdelhomenie) [inline calendar/debug.log]: Is this file needed for the tests ?

Comment (thomasdelhomenie) [reply calendar/debug.log]: @khouloudexo comment not considered

File: commons/src/main/java/org/exoplatform/platform/qa/ui/commons/Base.java (hunk 1 above)
Comment (thomasdelhomenie) [reply commons/src/main/java/org/exoplatform/platform/qa/ui/commons/Base.java]: @khouloudexo comment not considered

File: gatein/src/test/java/org/exoplatform/platform/qa/ui/gatein/GateinManagePagesTestIT.java (hunk 2 above)
Comment (thomasdelhomenie) [reply gatein/src/test/java/org/exoplatform/platform/qa/ui/gatein/GateinManagePagesTestIT.java]: @khouloudexo comment not considered

File: commons/src/main/java/org/exoplatform/platform/qa/ui/commons/Base.java (hunk 1 above)
Comment (aboughdiri) [reply commons/src/main/java/org/exoplatform/platform/qa/ui/commons/Base.java]: Done

File: gatein/src/test/java/org/exoplatform/platform/qa/ui/gatein/GateinManagePagesTestIT.java (hunk 2 above)
Comment (aboughdiri) [reply gatein/src/test/java/org/exoplatform/platform/qa/ui/gatein/GateinManagePagesTestIT.java]: Done

File: calendar/debug.log (hunk 5 above)
Comment (aboughdiri) [reply calendar/debug.log]: Done
